pkg/scalers/scalersconfig/typed_config.go

```diff
@@ -62,7 +62,7 @@ const (
 
 // separators for map and slice elements
 const (
-	elemSeparator       = ","
+	elemSeparator       = ",;"
```

Comment:
can you please make the `elemSeparator` configurable instead? I'd like this to be under the user's control to not restrict the available values much.

I can imagine the end usage something along the lines of
```go
type XYZ struct {
  Slice1 []string `keda:"name=slice1, order=triggerMetadata"`
  Slice2 []string `keda:"name=slice2, order=triggerMetadata, sep=;"`
}
```
default should remain `","`, because that would be difficult to configure by the end user.

you would need to add an entry to the enumeration of tags
https://github.com/kedacore/keda/blob/9e51a78c31f0d2c188ff01c5f84cb6816c0d1c51/pkg/scalers/scalersconfig/typed_config.go#L71-L78

and a new param
https://github.com/kedacore/keda/blob/9e51a78c31f0d2c188ff01c5f84cb6816c0d1c51/pkg/scalers/scalersconfig/typed_config.go#L82-L112

you can take an inspiration how `RangeSeparator` was added in [this commit](https://github.com/kedacore/keda/commit/25b46f6337db8a5e194d76cd93ba940767d83839#diff-734aa7734c44d2255c89b417f724c73b1d7a38d07fc06377174f759ad721f911)

Reply:
Thanks. I've made the `elemSeparator` configurable with the new parameter `separator` in TypedConfig. I have added in this commit: https://github.com/kedacore/keda/pull/6101/commits/51a17656c568ae0fe456acce0947d0e2c5270752